extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -566,4 +571,45 @@ private function getWikiaStatsData() {
 
 		return $wikiDetails['stats'];
 	}
+
+	private function getOtherLanguages( Title $title ) {
+		global $wgEnableLillyExt;
+
+		if ( empty( $wgEnableLillyExt ) ) {
+			return null;
+		}
+
+		$url = $title->getFullURL();
+		$url = str_replace( '.rychu.wikia-dev.com', '.wikia.com', $url );
```

Comment:
Is it necessary? Is it some kind of left over after development? What about other devboxes than `dev-rychu`? :)